cmd/monaco/download/download_configs.go

```diff
@@ -171,6 +181,19 @@ type downloadConfigsOptions struct {
 	onlyAutomation  bool
 }
 
+func (opts downloadConfigsOptions) valid() bool {
+	retVal := true
+	knownEndpoints := api.NewAPIs()
+	for _, e := range opts.specificAPIs {
+		if !knownEndpoints.Contains(e) {
+			log.Warn("unknown (or unsupported) classic endpoint with name %q provided via \"--api\" flag. More about supported classic endpoints can be found in documentation", e)
```

Comment:
this should be logged as an Error as we handle it as an error and don't continue if it happens